Document/0x06b-Basic-Security-Testing.md

```diff
@@ -186,9 +186,20 @@ iPhone:~ root#
 When accessing your iOS device via SSH consider the following:
 
 - The default users are `root` and `mobile`.
-- The default password is `alpine`.
+- The default password for both is `alpine`.
 
-Remember to change the default password for both users `root` and `mobile` as anyone being in the same network and knowing the IP address of your device an the well-know default password will be able to access your device as root.
+> Remember to change the default password for both users `root` and `mobile` as anyone on the same network can find the IP address of your device and connect via the well-know default password, which will give them root access to your device.
+
+If you forget your password and want to reset it to the default `alpine`:
+
+- Edit the file `/private/etc/master.password` on your jailbroken iOS device (using SSH over USB or an on-device shell as shown below)
```

Comment:
If you lost the password, you won't be using SSH to fix it, right ?

Reply:
xD good point!